packages/scss/src/components/inputs/checkbox/_checkbox.framed.scss

```diff
@@ -0,0 +1,11 @@
+%checkbox-framed {
+	@include fakeBorder(1px, _theme("commons.divider.color"));
+
+	display: flex;
```

Comment:
What about removing display flex and checkbox's right margin to use checkbox.mod-input which is implemented since 0.0.17 ? I don't know if it will be a use case but it would allow to use stacked and inline checkbox / radio / etc.

Reply:
I'd prefer to had a mod-stacked to the radio/checkbox-group that would change the flex-direction into column as it is more aligned with the philosophy of this design

Reply:
ok